lib/vdsm/virt/thinp.py

```diff
@@ -261,25 +261,19 @@ def _extend_drive_if_needed(self, drive):
         """
         Check if a drive should be extended, and start extension flow if
         needed.
-
-        We handle the drive according to the drive.threshold_state:
-
-        - UNSET: the drive needs to register for a new block threshold,
-                 so try to set it. We set the threshold both for chunked
-                 drives and non-chunked drives replicating to chunked
-                 drives.
-        - EXCEEDED: the drive needs extension, try to extend it.
-        - SET: this method should never receive a drive in this state,
-               emit warning and exit.
         """
         if drive.threshold_state == storage.BLOCK_THRESHOLD.SET:
+            # Should never happen; we monitor only UNSET and EXCEEDED drives.
```

Comment:
keep the same style of comments about the state - move it before `if` statement?

Reply:
`;` --> `:`?

Reply:
; is correct in this context.